src/customers/views/CustomerList/CustomerList.tsx

```diff
@@ -143,6 +144,18 @@ export const CustomerList: React.FC<CustomerListProps> = ({ params }) => {
 
   const handleSort = createSortHandler(navigate, customerListUrl, params);
 
+  React.useEffect(
+    () =>
+      navigate(
+        customerListUrl({
+          ...params,
+          ...DEFAULT_INITIAL_PAGINATION_DATA
+        }),
+        true
+      ),
+    [settings.rowNumber]
+  );
```

Comment:
LGTM :+1:, but can you extract it to hook, like `usePaginationReset(customerListUrl,settings.rowNumber)`?
It'd reduce code repetition making code DRY.